ee_plugin/ui/forms/add_ee_image.py

```diff
@@ -0,0 +1,101 @@
+import json
+from typing import Callable, Optional
+
+import ee
+from qgis.PyQt import QtWidgets
+from qgis.core import QgsMessageLog, Qgis
+
+from ...Map import addLayer
+from ..widgets import build_form_group_box, build_vbox_dialog
+from ..utils import call_func_with_values
+
+
+def form(accepted: Optional[Callable] = None, **dialog_kwargs) -> QtWidgets.QDialog:
+    """Display a dialog to add a GEE dataset to the QGIS map."""
+    dialog = build_vbox_dialog(
+        windowTitle="Add Google Earth Engine Image",
+        widgets=[
+            build_form_group_box(
+                title="Dataset",
```

Comment:
On `add_feature_collection`, we use "Source":

https://github.com/gee-community/qgis-earthengine-plugin/blob/9196a2f2ebbb390d8fb39c03b5c85cb423aff4b7/ee_plugin/ui/forms/add_feature_collection.py#L21